src/components/Board.js

```diff
@@ -1,33 +1,108 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import axios from 'axios';
-
 import './Board.css';
 import Card from './Card';
 import NewCardForm from './NewCardForm';
-import CARD_DATA from '../data/card-data.json';
+// import CARD_DATA from '../data/card-data.json';
 
 class Board extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
 
     this.state = {
       cards: [],
+      url: this.props.url,
+      boardName: this.props.boardName,
```

Comment:
You should only include things in `state` if they're going to change as your application runs. As far as I can tell `url` and `boardName` don't change, which means it would be clearer to leave them in `props`